src/components/Button/README.md

```diff
@@ -1,75 +1,146 @@
-Use a button any time the user should complete an action.
+A button indicates that a user can complete an action or make a choice, using a single click or tap. They are used for submitting forms, navigating to new pages, as well as toolbars and utilities.
 
 ```js
 const onClick = (event) => {
-  window.alert('🚨 Alarm sounded! 🚨');
+	window.alert('🚨 Alarm sounded! 🚨');
```

Comment:
😮 TABS!

I'll need to get some kind of lint setup for this. It's fine for now.